### PR TITLE
CircleCI migration

### PR DESCRIPTION
**Pull request by frosario**

### CODE REVIEW

File: packages/styles/globals.import.styl [hunk 1]
```diff
@@ -117,6 +117,11 @@ a
     top auto
     width 100%
```
Comment (aslagle) [inline packages/styles/globals.import.styl]: should we remove the css for the old toast position?

Comment (frosario) [inline packages/styles/globals.import.styl]: Looks like it is still referenced in a coffee file:
`git grep toast-top-right-custom` results in:
packages/controllers/toastr.coffee:  positionClass: 'toast-top-right-custom'
packages/styles/globals.import.styl:.toast-top-right-custom

Comment (frosario) [inline packages/styles/globals.import.styl]: Removed that unused class in commit: 53c200a59d816c44cd702416bd9bb22e8f2975b5

**Comment by aslagle** [comment]
:+1:

**Pull request merged**